raiden/network/transport/matrix/client.py

```diff
@@ -111,10 +115,13 @@ def __init__(
     ) -> None:
         super().__init__(*args, **kwargs)
 
+        self.server_ident: Optional[str] = None
+
         http_adapter = HTTPAdapter(pool_maxsize=pool_maxsize)
         https_adapter = HTTPAdapter(pool_maxsize=pool_maxsize)
         self.session.mount("http://", http_adapter)
         self.session.mount("https://", https_adapter)
+        self.session.hooks["response"].append(self._record_server_ident)
```

Comment:
What does this do exactly? Every time it receives a response `self.server_ident` is updated?

Reply:
Yes, it could be removed after the first time. But otoh it doesn't really cost much. 

The server ident is used later on in a log message, just to get more debugging info in case someone reports an issue with the transport.